sound/soc/sof/amd/acp-ipc.c

```diff
@@ -130,6 +130,13 @@ static void acp_dsp_ipc_get_reply(struct snd_sof_dev *sdev)
 		memcpy(msg->reply_data, &reply, sizeof(reply));
 		ret = reply.error;
 	} else {
+		/*
+		 * return when message reply_size is Zero.
+		 * to support no_reply ipc tx messages
+		 */
+		if (!msg->reply_size)
+			goto out;
+
```

Comment:
the commit message is confusing, it's not about a specific API but about supporting an IPC with a reply_size set to zero. The no_reply IPC is just a wrapper.

Reply:
Okay, will change.